src/backend/main.ts

```diff
@@ -468,7 +468,7 @@ ipcMain.on('unlock', () => {
     unlinkSync(join(gamesConfigPath, 'lock'))
     if (powerId) {
       logInfo('Stopping Power Saver Blocker', LogPrefix.Backend)
-      return powerSaveBlocker.stop(powerId)
+      powerSaveBlocker.stop(powerId)
```

Comment:
I had to change this cause typescript starts complaining about this function with electron 26 (not all code paths returned a value)